impl/pkg/storage/storage.go

```diff
@@ -9,8 +9,8 @@ import (
 	"github.com/sirupsen/logrus"
 
 	"github.com/TBD54566975/did-dht/pkg/dht"
-	"github.com/TBD54566975/did-dht/pkg/storage/db/bolt"
 	"github.com/TBD54566975/did-dht/pkg/storage/db/postgres"
+	"github.com/TBD54566975/did-dht/pkg/storage/db/sqlite"
 )
 
 type Storage interface {
```

Comment:
wondering if there should be a set of dynamic dispatch methods over the storage calls that can do a common set of validations (empty checks, for example)